pages/implementations/earl-reports.md

```diff
@@ -71,13 +71,44 @@ The following properties are required for each Assertion:
 			"assertions": [
 				{
 					"@type": "Assertion",
-					"test": { "title": "My Tool's rule title" },
-					"result": { "outcome": "earl:passed" }
+					"result": { "outcome": "earl:passed" },
+					"test": { ... },
 				}
 			]
 		}
 	]
 }
 ```
 
+### Test Criterion
+
+For each result, we'll need to know which rule or procedure in the tool reported it, and which WCAG success criteria were failed. To do so, on the `test` property add an object. This must include:
+
+- `title`: A string, the uniquely identifies what rule or procedure in the tool found the result.
+- `isPartOf`: An array of strings, listing the WCAG success criteria that fail when the rule or procedure fails. These must use the IDs of the WCAG success criteria, prefaced with `WCAG2:`. For example `WCAG2:name-role-value`.
```

Comment:
Any chance this will bite us if we don't use 2.0, 2.1, etc? I would hope no significant changes in existing SC before WCAG 3.